composites/Plugin/SocialPreviews/Facebook/components/FacebookImage.js

```diff
@@ -0,0 +1,154 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import {
+	determineFacebookImageProperties,
+	LANDSCAPE_HEIGHT,
+	LANDSCAPE_WIDTH,
+	PORTRAIT_HEIGHT,
+	PORTRAIT_WIDTH,
+	SQUARE_HEIGHT,
+	SQUARE_WIDTH,
+} from "../helpers/determineFacebookImageProperties";
+import colors from "../../../../../style-guide/colors.json";
```

Comment:
generally, we should try to keep external and internal dependencies separated in two blocks with comments.